### DISCUSSION TITLE
Add Argentina prefix (+54) as an option for sms message regarding 2FA

### DISCUSSION THREAD

**Discussion by vituchon** [community/community#140817] [Other Feature Feedback, Questions, & Ideas]
Hi There! Hope you are well!

I do need an option for Argentina Here

![image](https://github.com/user-attachments/assets/bfb777cd-0da4-4020-934f-92dda41d35ac)

![image](https://github.com/user-attachments/assets/08400044-bf32-4932-b778-83511cf2ba9e)

Please can you add it?

Thanks in advance.

Greetings
Vïctor.

**Comment by queenofcorgis** [comment]
Hey there! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.

You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by vituchon** [reply]
Thanks @queenofcorgis .

If you don't mind telling me what would be the correct category I won't mind to re send this issue to the proper people that can address this matter. Do note that I wasn't sure where to place this ticket so I aimed to this category in first place without being sure.

Thanks again.
Have a good day.

**Reply by felobadino** [reply]
@queenofcorgis I'm seeing that you're answering the same thing in every post on this topic, regardless of the country missing from the SMS 2FA process, but I'd still like you (or GitHub Support Team) to take the time to include my country (Argentina +54) and the remaining ones in the SMS process for 2FA, giving the topic complete versatility and not making us dependent on third-party apps. Thanks in advance.
